### PR TITLE
expanded text on RDF vocabs

### PR DESCRIPTION
**Pull request by pmcb55**
# Chore
Just updates the README text to explain the vocab modules we publish to NPM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,12 +17,56 @@ Webpack, Rollup, or Parcel.
...
+systems to share concepts and ideas).
+
+There are already many common vocabularies published on the web, such as the
+[vCard vocabulary](https://www.w3.org/TR/vcard-rdf/) for describing the concepts
+and ideas associated with People and Organizations, or [Schema.org](https://schema.org/)
+for describing the things search engines are typically interested in, intended
+to be used by webmasters to mark up their pages in ways that those search
+engines can then understand.
+
+As a convenience for JavaScript developers, Inrupt publishes various NPM modules
```
Comment (Vinnl) [inline README.md:40]:
```suggestion
As a convenience for JavaScript developers, Inrupt publishes various npm modules
```

File: README.md [hunk 2]
```diff
@@ -17,12 +17,56 @@ Webpack, Rollup, or Parcel.
...
+This module bundles together JavaScript classes representing all the RDF
+vocabularies related to Solid (e.g. Solid terms, WebACL, ACP, etc.): [@inrupt/vocab-solid-common](https://www.npmjs.com/package/@inrupt/vocab-solid-common)
+
+### Inrupt-specific RDF vocabularies
+
+This module bundles together JavaScript classes representing all the RDF
+vocabularies created and maintained by Inrupt (e.g. an Inrupt test vocabulary,
+Inrupt glossaries, Inrupt product vocabularies, etc.): [@inrupt/vocab-inrupt-common](https://www.npmjs.com/package/@inrupt/vocab-inrupt-common)
+
+**NOTE:** All the above NPM modules are currently generated and published
```
Comment (Vinnl) [inline README.md:67]: (Spelling "NOTE" in all-caps will have that read out as an acronym by screen readers, I believe.)

```suggestion
**Note:** All the above npm modules are currently generated and published
```

File: README.md [hunk 3]
```diff
@@ -17,12 +17,56 @@ Webpack, Rollup, or Parcel.
...
+to NPM automatically by a closed-source Inrupt tool. We plan to open-source that
```
Comment (Vinnl) [inline README.md:68]:
```suggestion
to npm automatically by a closed-source Inrupt tool. We plan to open-source that
```

**Review by Vinnl — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (pmcb55) [reply README.md]: Sure thing, although by the same token won't having 'npm' in lowercase mean it won't be read out as an acronym by screen readers (meaning it'll become 'nipem', or something :) )...?!

Comment (Vinnl) [reply README.md]: Yeah good point, blame npm and their stupid name capitalisation policy for that I suppose :)

**Pull request merged**